touchforms/formplayer/static/formplayer/script/test_helpers/jasmine-fixture.js

```diff
@@ -0,0 +1,433 @@
+/* jasmine-fixture - 1.2.2
```

Comment:
cool, probably not need here since it's only loaded in tests, but that's definitely something for the `underscore.js` file